<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1121</width>
    <height>729</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="font">
   <font>
    <family>Times New Roman</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>平面有限元</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1121</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="File">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="Edit">
    <property name="title">
     <string>编辑</string>
    </property>
   </widget>
   <widget class="QMenu" name="Tool">
    <property name="title">
     <string>工具</string>
    </property>
   </widget>
   <widget class="QMenu" name="Postprocessing">
    <property name="title">
     <string>后处理</string>
    </property>
    <addaction name="actionResult"/>
   </widget>
   <widget class="QMenu" name="Window">
    <property name="title">
     <string>窗口</string>
    </property>
   </widget>
   <widget class="QMenu" name="Help">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionHelp"/>
   </widget>
   <widget class="QMenu" name="Part">
    <property name="title">
     <string>部件</string>
    </property>
    <addaction name="actionNewPart"/>
    <addaction name="actionPartManager"/>
   </widget>
   <widget class="QMenu" name="Material">
    <property name="title">
     <string>材料</string>
    </property>
    <addaction name="actionNewMaterial"/>
    <addaction name="actionMaterialManager"/>
   </widget>
   <widget class="QMenu" name="Interaction">
    <property name="title">
     <string>边界条件</string>
    </property>
    <addaction name="actionNewBoundaryCondition"/>
    <addaction name="actionBoundaryConditionManager"/>
   </widget>
   <widget class="QMenu" name="Load">
    <property name="title">
     <string>荷载</string>
    </property>
    <addaction name="actionNewLoad"/>
    <addaction name="actionLoadManager"/>
   </widget>
   <widget class="QMenu" name="Data">
    <property name="title">
     <string>数据</string>
    </property>
    <addaction name="actionElementNodeNumber"/>
    <addaction name="actionNodeCoordinate"/>
    <addaction name="actionFixedDegree"/>
    <addaction name="separator"/>
    <addaction name="actionEKTest"/>
    <addaction name="actionLoad"/>
    <addaction name="separator"/>
    <addaction name="actionID"/>
    <addaction name="actionLM"/>
    <addaction name="actionFirstNomzeroRow"/>
    <addaction name="actionAssistentMatrix"/>
    <addaction name="separator"/>
    <addaction name="actionNodeDisplacement"/>
   </widget>
   <widget class="QMenu" name="Job">
    <property name="title">
     <string>作业</string>
    </property>
    <addaction name="actionJob"/>
   </widget>
   <widget class="QMenu" name="Analyze">
    <property name="title">
     <string>分析</string>
    </property>
    <addaction name="actionAnalyze"/>
   </widget>
   <addaction name="File"/>
   <addaction name="Edit"/>
   <addaction name="Tool"/>
   <addaction name="Part"/>
   <addaction name="Material"/>
   <addaction name="Interaction"/>
   <addaction name="Load"/>
   <addaction name="Job"/>
   <addaction name="Analyze"/>
   <addaction name="Postprocessing"/>
   <addaction name="Data"/>
   <addaction name="Window"/>
   <addaction name="Help"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="actionNewPart"/>
   <addaction name="actionNewMaterial"/>
   <addaction name="actionNewBoundaryCondition"/>
   <addaction name="actionNewLoad"/>
   <addaction name="actionJob"/>
   <addaction name="actionAnalyze"/>
  </widget>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/icons/332.bmp</normaloff>:/icons/icons/332.bmp</iconset>
   </property>
   <property name="text">
    <string>帮助</string>
   </property>
   <property name="toolTip">
    <string>帮助</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/icons/exit.bmp</normaloff>:/icons/icons/exit.bmp</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionNewPart">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/images/images/Part.png</normaloff>:/images/images/Part.png</iconset>
   </property>
   <property name="text">
    <string>新建部件</string>
   </property>
   <property name="toolTip">
    <string>新建部件</string>
   </property>
  </action>
  <action name="actionPartManager">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/icons/413.bmp</normaloff>:/icons/icons/413.bmp</iconset>
   </property>
   <property name="text">
    <string>部件管理器</string>
   </property>
   <property name="toolTip">
    <string>部件管理器</string>
   </property>
  </action>
  <action name="actionNewMaterial">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/images/images/Material.png</normaloff>:/images/images/Material.png</iconset>
   </property>
   <property name="text">
    <string>新建材料</string>
   </property>
   <property name="toolTip">
    <string>新建材料</string>
   </property>
  </action>
  <action name="actionMaterialManager">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/icons/413.bmp</normaloff>:/icons/icons/413.bmp</iconset>
   </property>
   <property name="text">
    <string>材料管理器</string>
   </property>
   <property name="toolTip">
    <string>材料管理器</string>
   </property>
  </action>
  <action name="actionNewBoundaryCondition">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/images/images/Interaction.png</normaloff>:/images/images/Interaction.png</iconset>
   </property>
   <property name="text">
    <string>新建边界条件</string>
   </property>
   <property name="toolTip">
    <string>新建边界条件</string>
   </property>
  </action>
  <action name="actionBoundaryConditionManager">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/icons/413.bmp</normaloff>:/icons/icons/413.bmp</iconset>
   </property>
   <property name="text">
    <string>边界条件管理器</string>
   </property>
   <property name="toolTip">
    <string>边界条件管理器</string>
   </property>
  </action>
  <action name="actionNewLoad">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/images/images/Load.png</normaloff>:/images/images/Load.png</iconset>
   </property>
   <property name="text">
    <string>新建荷载</string>
   </property>
   <property name="toolTip">
    <string>新建荷载</string>
   </property>
  </action>
  <action name="actionLoadManager">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/icons/413.bmp</normaloff>:/icons/icons/413.bmp</iconset>
   </property>
   <property name="text">
    <string>荷载管理器</string>
   </property>
   <property name="toolTip">
    <string>荷载管理器</string>
   </property>
  </action>
  <action name="actionElementNodeNumber">
   <property name="text">
    <string>单元信息</string>
   </property>
   <property name="toolTip">
    <string>单元信息</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>荷载列阵</string>
   </property>
   <property name="toolTip">
    <string>荷载列阵</string>
   </property>
  </action>
  <action name="actionNodeCoordinate">
   <property name="text">
    <string>结点信息</string>
   </property>
   <property name="toolTip">
    <string>结点信息</string>
   </property>
  </action>
  <action name="actionFixedDegree">
   <property name="text">
    <string>约束自由度</string>
   </property>
   <property name="toolTip">
    <string>约束自由度</string>
   </property>
  </action>
  <action name="actionID">
   <property name="text">
    <string>结点位移信息</string>
   </property>
   <property name="toolTip">
    <string>结点位移信息</string>
   </property>
  </action>
  <action name="actionLM">
   <property name="text">
    <string>连接数组</string>
   </property>
   <property name="toolTip">
    <string>连接数组</string>
   </property>
  </action>
  <action name="actionJob">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/images/images/job.ico</normaloff>:/images/images/job.ico</iconset>
   </property>
   <property name="text">
    <string>作业参数</string>
   </property>
   <property name="toolTip">
    <string>作业参数</string>
   </property>
  </action>
  <action name="actionAssistentMatrix">
   <property name="text">
    <string>辅助数组</string>
   </property>
   <property name="toolTip">
    <string>辅助数组</string>
   </property>
  </action>
  <action name="actionFirstNomzeroRow">
   <property name="text">
    <string>每列首个非零元素索引</string>
   </property>
  </action>
  <action name="actionEKTest">
   <property name="text">
    <string>单元刚度矩阵测试</string>
   </property>
   <property name="toolTip">
    <string>单元刚度矩阵测试</string>
   </property>
  </action>
  <action name="actionAnalyze">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/images/images/Analyze.ico</normaloff>:/images/images/Analyze.ico</iconset>
   </property>
   <property name="text">
    <string>运行分析</string>
   </property>
   <property name="toolTip">
    <string>运行分析</string>
   </property>
  </action>
  <action name="actionNodeDisplacement">
   <property name="text">
    <string>结点位移</string>
   </property>
  </action>
  <action name="actionResult">
   <property name="text">
    <string>结果输出</string>
   </property>
   <property name="toolTip">
    <string>结果输出</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>504</x>
     <y>307</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
